### PR TITLE
Support brute force search on MacOS.

### PR DESCRIPTION
**Pull request by soothing-rain**

### CODE REVIEW

File: internal/core/src/segcore/collection_c.cpp [hunk 1]
```diff
@@ -24,6 +29,9 @@ void
 DeleteCollection(CCollection collection) {
     auto col = (milvus::segcore::Collection*)collection;
     delete col;
+#ifndef __APPLE__
```
Comment (jaime0815) [inline internal/core/src/segcore/collection_c.cpp:32]: malloc_trim() should be enabled on linux

File: internal/core/cmake/Utils.cmake [hunk 2]
```diff
@@ -72,6 +72,21 @@ MACRO(get_milvus_version)
     endif()
 ENDMACRO(get_milvus_version)
 
+# get knowhere version
+MACRO(get_knowhere_version)
```
Comment (jaime0815) [inline internal/core/cmake/Utils.cmake:76]: due to duplicated file name with knowhere project, get_knowhere_version command cann't be found.

Comment (jaime0815) [reply internal/core/cmake/Utils.cmake:76]: this is a knohwere bug

File: internal/core/src/segcore/collection_c.cpp (hunk 1 above)
Comment (soothing-rain) [reply internal/core/src/segcore/collection_c.cpp:32]: Right. It is only disabled on MacOS. (ifndef)

File: internal/core/cmake/Utils.cmake (hunk 2 above)
Comment (soothing-rain) [reply internal/core/cmake/Utils.cmake:76]: I see. Any fix suggestion? Renaming either one of the cmake file?

Comment (jaime0815) [reply internal/core/cmake/Utils.cmake:76]: I will rename this file of knowhere to fix it.

Comment (jaime0815) [reply internal/core/cmake/Utils.cmake:76]: Please revert get_knowhere_version

**Pull request merged**